doc/content/guides/connecting-gateways/tektelickonamicro/_index.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Tektelic Kona Micro IoT LoRaWAN Gateway"
+description: ""
+weight: 1
+---
+
+Tektelic Kona Micro IoT LoRaWAN Gateway is an 8 Channel LoRaWAN gateway, whose technical specifications can be found in [the official documentation](https://tektelic.com/wp-content/uploads/KONA-Micro.pdf). This page guides you to connect it to {{% tts %}}.
```

Comment:
Small cap channel, ethernet, internet, etc

Reply:
`Internet` (meaning the World Wide Web) is grammatically capitalised, as opposed to `internet` which is just a connection of any networks.